tests/unit/test_paramfile.py

```diff
@@ -62,6 +63,22 @@ def test_no_match_uris_returns_none(self):
     def test_non_string_type_returns_none(self):
         self.assertIsNone(get_paramfile(100))
 
+    def test_get_file_utf8_text(self):
+        contents = b'\xe9\x96\xa2\xe6\x88\xb8\xe3\x81\xa6\xe3\x81\x99\xe3\x81\xa8'
+        filename = self.files.create_file('foo', contents, mode='wb')
+        data = get_file('', filename, 'r')
+        self.assertEqual(data, u'\u95a2\u6238\u3066\u3059\u3068')
+        self.assertIsInstance(data, six.string_types)
+
+    @unittest.skipIf(locale.getpreferredencoding() != 'cp932',
+            'encoding fallback test with cp932 system')
+    def test_get_file_fallback_platformdefault_in_cp932(self):
```

Comment:
This test will almost never be run. I would prefer to patch out the preferred encoding so that we can have this test be run every time the test suite is run.